package.json

```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/yannickcr/eslint-plugin-react",
   "bugs": "https://github.com/yannickcr/eslint-plugin-react/issues",
   "devDependencies": {
-    "babel-eslint": "5.0.0-beta6",
+    "babel-eslint": "5.0.0-beta9",
```

Comment:
@baer yes that was intentional. When running test via `npm test` i hit this issue.

```
➜  eslint-plugin-react git:(master) ✗ npm run test

> eslint-plugin-react@3.16.1 test /Users/nkatak1/devr/eslint-plugin-react
> npm run lint && npm run unit-test


> eslint-plugin-react@3.16.1 lint /Users/nkatak1/devr/eslint-plugin-react
> eslint ./


> eslint-plugin-react@3.16.1 unit-test /Users/nkatak1/devr/eslint-plugin-react
> istanbul cover --dir reports/coverage node_modules/mocha/bin/_mocha tests/**/*.js -- --reporter dot



  ․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․TypeError: Cannot read property 'visitClass' of undefined
    at monkeypatch (/Users/nkatak1/devr/eslint-plugin-react/node_modules/babel-eslint/index.js:220:40)
    at Object.exports.parse (/Users/nkatak1/devr/eslint-plugin-react/node_modules/babel-eslint/index.js:362:5)
    at parse (/Users/nkatak1/devr/eslint-plugin-react/node_modules/eslint/lib/eslint.js:574:27)
    at EventEmitter.module.exports.api.verify (/Users/nkatak1/devr/eslint-plugin-react/node_modules/eslint/lib/eslint.js:701:19)
    at runRuleForItem (/Users/nkatak1/devr/eslint-plugin-react/node_modules/eslint/lib/testers/rule-tester.js:315:38)
    at testValidTemplate (/Users/nkatak1/devr/eslint-plugin-react/node_modules/eslint/lib/testers/rule-tester.js:333:26)
    at Context.<anonymous> (/Users/nkatak1/devr/eslint-plugin-react/node_modules/eslint/lib/testers/rule-tester.js:421:25)
    at callFn (/Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runnable.js:286:21)
    at Test.Runnable.run (/Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runnable.js:279:7)
    at Runner.runTest (/Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runner.js:421:10)
    at /Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runner.js:528:12
    at next (/Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runner.js:341:14)
    at /Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runner.js:351:7
    at next (/Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runner.js:283:14)
    at Immediate._onImmediate (/Users/nkatak1/devr/eslint-plugin-react/node_modules/mocha/lib/runner.js:319:5)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

upon lookup I found that updating eslint-babel dependency fixed the issue. So updated that dependency.